pom.xml

```diff
@@ -629,16 +638,20 @@
         </profile>
 
         <profile>
-            <id>postgresql11</id>
```

Comment:
Seems we're replacing the PG 11 profile by the cockroach one. Wouldn't it make more sense to replace the 9.6 (somewhat obsolete) one instead ?

Reply:
Here we are removing specific profiles for each database engine. Giving flexibility to define engine version (docker image) throw maven parameter.

Reply:
My point was that <image.postgresql>postgres:9.6.14</image.postgresql> points to an obsolete driver, the default for the "postgres" profile should be 11.

Reply:
Didn't understand the main issue that you point. Your right, and in that case, de default image should be latest, instead of a "today latest x.y.z version".

Reply:
End of life for PostgreSQL is in 2021, and we support it in our products.
But the reason the postgresql11 profile was removed is that now it is possible to choose the Docker image for tests.
For Postgres 11, the flags could be `-Ppostgresql -Dimage.postgresql=postgres:11.4`, and similarly we could choose an image for Postgres 10 or even 12.

Reply:
PS: just noticed the comments from Mario :smile: 
To clarify, the reason for removing postgresql11 is that we can use it, or even a newer version, using the image version flag; the reason for keeping 9.6 as the default is that both PostgreSQL and us in our products still support that version (and we don't want to introduce support for new features from v11.x that won't work in the older v9.x)

Reply:
I don't agree that the version for the "postgres" profile is PG 9 (the one that people use without explicitly overriding the image) but I'm not blocking this review for this.

Reply:
Oops, sorry, was confusing with Oracle ...